content/en/blog/2020-05-14-configuring-routing-for-metallb-in-l2-mode.md

```
@@ -0,0 +1,189 @@
+ip rule add from 1.2.3.0/24 lookup 100
+ip rule add from 1.2.3.0/24 to 10.112.0.0/12 lookup main
+```
+
+Where:
+
+- `1.2.3.0/24` — is external network
+- `10.112.0.0/12` — is your podNetwork
+
+The second rule must be added after the first one in order to get the highest priority.
+
+**Method with connection marking**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)